src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -753,6 +754,14 @@ public function testDoesntHaveIssuesWithUnionConstTypes()
             $this->markTestSkipped('phpstan/phpdoc-parser required for this test');
         }
 
+        $versionRange = InstalledVersions::getVersionRanges('symfony/property-info');
+
+        if (
+            '5.4.x-dev' !== $versionRange && version_compare($versionRange, '5.4.47', '<')
+            || version_compare($versionRange, '5.4.48', '>=') && '6.4.x-dev' !== $versionRange && version_compare($versionRange, '6.4.15', '<')) {
```

Comment:
not sure I get the contraint on 5.4.48, shouldn't this be on >= 6?
```suggestion
            || version_compare($versionRange, '6', '>=') && '6.4.x-dev' !== $versionRange && version_compare($versionRange, '6.4.15', '<')
        ) {
```